### PR TITLE
Remove snowcrash xcode workspace

### PR DESCRIPTION
**Pull request by pksunkara**
With the introduction of workspace in drafter which contains this snowcrash xcodeproj, we don't need the workspace here anymore.

### CODE REVIEW

File: ext/markdown-parser [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit ff1e1cd7349d17e8381842cc3c3037f34ad19c8e
+Subproject commit 63ca9e3323b80901156d03e1280cee06928b1185
```
Comment (kylef) [inline ext/markdown-parser]: Unrelated change?

**Comment by kylef** [comment]
I don't think If follow why this workspace has anything to do with a workspace in drafter.

This project should work without drafter, drafter is a dependant of this project.

File: ext/markdown-parser (hunk 1 above)
Comment (pksunkara) [inline ext/markdown-parser]: Nope. Related since that commit in markdown-parser updates markdown-parser xcodeproj.

**Comment by pksunkara** [comment]
Is that how workspaces are treated? I thought workspaces are about having all the xcodeproj together.

**Comment by kylef** [comment]
@pksunkara Nevermind, I though you was referring to the internal workspace within the project.

This project doesn't have one, it must've been created before they we're a thing (Xcode 3).
